near/omni-bridge/src/lib.rs

```diff
@@ -561,7 +565,12 @@ impl Contract {
         self.remove_promise(&message_storage_account_id);
 
         if response.is_ok() {
-            self.init_transfer_internal(transfer_message, message_storage_account_id, storage_owner)
+            self.pay_native_fee_from_message_account(
```

Comment:
We can't panic here. I suggest to make `pay_native_fee_from_message_account` return result instead